src/styles/ftva/_divider-way-finder.scss

```diff
@@ -0,0 +1,5 @@
+.ftva.divider-way-finder {
+  &.color-ftva {
```

Comment:
instead of .color-ftva, make the &.color-default styles  the new `--color-border: #{$accent-blue}` styles

Reply:
I see now.